services/app-web/src/common-code/data/statePrograms.json

```diff
@@ -994,6 +1009,41 @@
                     "id": "6b6afeaf-aeff-45f7-a14c-992a03fa372f",
                     "fullName": "Children's CHIP Dental Services",
                     "name": "CHIP Dental "
+                },
+                {
+                    "id": "",
+                    "fullName": "",
```

Comment:
Huh empty rows being brought in 🔍 taking a look now at the actual spreadsheet to see what happened